cmd/create.go

```diff
@@ -128,21 +129,37 @@ var createCmd = &cobra.Command{
 		}
 
 		project.AbsolutePath = currentWorkingDir
-
-		// This calls the templates
+		spinStatus := make(chan struct{})
```

Comment:
I think the reason is we dont actually pass this channel.
This is not properly listening to the status of the spinner, so on line 149 we are listening but are not getting any value
indicating the the spinner status completed.

Reply:
I'm not sure why I left this here, it was here in the original implementation and kind of left it. removed it now.

Reply:
Nevermind, I figured out what is wrong.